shell/app/modules/application/pages/repo/components/repo-editor.tsx

```diff
@@ -162,6 +163,11 @@ const RepoEditor = ({
           maxLength={255}
           onChange={(e) => {
             updater.fileName(e.target.value.trim());
+            if (formRef.current) {
+              formRef.current.setFieldsValue({
```

Comment:
1. formRef.current?.setFieldsValue instead
2. if user has already input something in the `message` field. this will override the user input. we should check `message` field is untouched first

Reply:
User would not change file name again usually, even override does not matter.

Reply:
What i mean is, maybe user would not edit file name firstly, instead, fill in the content and input the commit message `my commit purpose`. Finally notice the file name is blank. then input the file name but find the written commit message is gone.

Reply:
I understand what you mean, but when edit file name, will auto change message and set touched flag to true, which means after typed the first letter, the message will not update again to sync with file name.
Have added autoFocus to filename input to make sure user type filename first.